uint16 penalty             #// penalty state of the player
uint16 secsTillUnpenalised # // estimate of time till unpenalised
